benchflow-test-manager/application/src/test/resources/data/exploration/one-at-a-time/users-memory-environment/benchflow-test.yml

```diff
@@ -16,13 +16,13 @@ configuration:
       camunda:
         resources:
           memory:
```

Comment:
It was a mistake to test for ranges because it generates too many experiments - detailed tests are in the DSL.